### PR TITLE
Fixed incompatibilities with integration tests

### PR DESCRIPTION
**Pull request by CrochetFeve0251**
## Description

Fix compatibilities errors in tearDown and setUp function from Integration tests that prevent tests from running.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Is the solution different from the one proposed during the grooming?

Yes as there is no grooming

## How Has This Been Tested?

- [ ] Automated Tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/Integration/AdminTestCase.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ public function set_up() {
 		error_reporting( $this->error_level & ~E_WARNING );
 	}
 
-	public function tear_down() {
+	public function tear_down(): void {
```
Comment (remyperona) [inline tests/Integration/AdminTestCase.php]: The `tear_down()` method doesn't need the typehint

File: tests/Integration/inc/Engine/Cache/AdminSubscriber/addWpCacheStatusTest.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
 class Test_AddWpCacheStatusTest extends AdminTestCase {
 	protected $path_to_test_data = '/inc/Engine/Cache/WPCache/addWpCacheStatusTest.php';
 
-	public function tearDown() {
+	public function tearDown(): void {
 		remove_filter( 'rocket_set_wp_cache_constant', [ $this, 'return_false' ] );
 
 		parent::tearDown();
```
Comment (remyperona) [inline tests/Integration/inc/Engine/Cache/AdminSubscriber/addWpCacheStatusTest.php]:
```suggestion
	public function tear_down() {
		remove_filter( 'rocket_set_wp_cache_constant', [ $this, 'return_false' ] );

		parent::tear_down();
```

**Review by remyperona — CHANGES_REQUESTED** [review]
The only file that needs changing is `addWpCacheStatusTest`

File: tests/Integration/AdminTestCase.php (hunk 1 above)
Comment (CrochetFeve0251) [reply tests/Integration/AdminTestCase.php]: @Tabrisrp I will  try to change this one first

Comment (CrochetFeve0251) [reply tests/Integration/AdminTestCase.php]: Ok it works just with `addWpCacheStatusTest`

**Review by remyperona — APPROVED** [review]

**Pull request merged**